README.md

```diff
@@ -112,7 +117,29 @@ while ... { ... }
 ```
 Invariants must have names (for now).
 
-Finally, there is a `variant` expression, which may be useful when defining *logical functions*, whose termination must be proved. You can give it an expression as argument, whose values must strictly decrease (in a known well-founded order) at each recursive call.
+Finally, there is a `variant` expression, which may be useful when defining *logical functions*, whose termination must be proved. You can give it an expression as argument, whose value must strictly decrease (in a known well-founded order) at each recursive call.
+
+## Controlling verification
+
+We also have features for controlling verification.
+
+First, the `trusted` marker lets Creusot trust the implementation and specs.
+More specifically, you can put `#[trusted]` on a function like the following:
+```rust
+#[trusted]
+#[ensures(result == 42u32)]
+fn the_answer() -> u32 {
+  trusted_super_oracle("the answer to life, the universe and everything")
+}
+```
+
+Also, we have the `UNBOUNDED` option.
```

Comment:
this only works in tests. When running `creusot-rustc` you actually want to set the environment variable `CREUSOT_UNBOUNDED=1`